GUI/GUITextPanel.h

```diff
@@ -302,4 +320,4 @@ class GUITextPanel :
 }; // namespace RTE
 
 
-#endif  //  _GUITEXTPANEL_
\ No newline at end of file
+#endif  //  _GUITEXTPANEL_
```

Comment:
Github editor refuses to not have this new line and I don't wanna set this thing up properly.